<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni_win</class>
 <widget class="QDialog" name="omni_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Omnivore</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Choose your Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="10" column="1">
       <widget class="QSpinBox" name="health_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="size_l">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSlider" name="strength_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="eyesight_l">
        <property name="text">
         <string>Eye Sight</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="strength_l">
        <property name="text">
         <string>Strength</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="visibility_l">
        <property name="text">
         <string>Visibility</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QSlider" name="health_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="strength_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="energy_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSlider" name="energy_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="eyesight_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="health_l">
        <property name="text">
         <string>Health</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSlider" name="eyesight_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="size_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="energy_l">
        <property name="text">
         <string>Energy</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="visibility_c">
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QSlider" name="visibility_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QSlider" name="size_s">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QPushButton" name="add_omnitrix">
        <property name="text">
         <string>Add Omnivore</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
